### DISCUSSION TITLE
AuthenticationError: The email or password provided is incorrect.

### DISCUSSION THREAD

**Discussion by rnwcode** [payloadcms/payload#677] [General]
# Bug Report

I get the following error message when trying to log in. Both via the dashboard and purely via the Rest API.  

```
[21:36:34] ERROR (payload): AuthenticationError: The email or password provided is incorrect.
    at new ExtendableError (/var/www/xxx/node_modules/payload/src/errors/APIError.ts:26:11)
    at new APIError (/var/www/xxx/node_modules/payload/src/errors/APIError.ts:43:5)
    at new AuthenticationError (/var/www/xxx/node_modules/payload/src/errors/AuthenticationError.ts:6:5)
    at login (/var/www/xxx/node_modules/payload/src/auth/operations/login.ts:93:11)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at loginHandler (/var/www/xxx/node_modules/payload/src/auth/requestHandlers/login.ts:8:20)
```

The email configuration looks like this: 

```
payload.init({
  secret: process.env.PAYLOAD_SECRET,
  mongoURL: process.env.MONGODB_URI,
  express: app,
  onInit: () => {
    payload.logger.info(`Payload Admin URL: ${payload.getAdminURL()}`)
  },
  email: {
    transportOptions: {
      host: 'xxx',
      auth: {
        user: 'xxx',
        pass: 'xxx'
      },
      port: 587,

    },
    fromName: 'xxx',
    fromAddress: 'xxx',
    logMockCredentials: true
  }
})
```

The Collection is structured like this: 

```
const Users: CollectionConfig = {
  slug: 'users',
  auth: {
    tokenExpiration: 31104000,
    verify: false,
    maxLoginAttempts: 100,
  },
  admin: {
    useAsTitle: 'email',
  },
  access: {
    read: () => true,
    create: () => true,
    update: () => true
  },
 ...
}
```

Unfortunately, it is not clear where exactly the error should be, also I could not find anything in the documentation.

**Comment by jmikrut** [comment]
Hey @rnwcode — do you have the ability to inspect your database and look at how many login attempts you've made? Look directly at the user record that you're trying to log in with.

If you're 100% positive that your email and password are correct, it's possible that the login attempts are over your limit of 100 and you are now locked from being able to log in.

Another thing you could do is just create a brand new user, and try logging in with that user.

I don't think this is a bug with Payload, so I'm going to move this issue to a Discussion instead for now but we can keep the conversation going there.

**Comment by jtsiaperas** [comment]
I had a similar problem and discovered it was because I was using a middleware. I had it set to run on every route so it was interfering with whatever payload uses for the authentication. I changed it to a specific route only and it works now. Here is the offending line of code:
app.use(formidableMiddleware({uploadDir: './media'}));

and the fix was putting it in the upload route instead:
app.post('/upload', formidableMiddleware({uploadDir: './media'}), (req, res) => {
